### PR TITLE
Update multiple seller FLEDGE testing explainer

### PR DESCRIPTION
**Pull request by rahulkooverjee-google**
- Update API details 
- Share link to official GPT developer documentation

### CODE REVIEW

File: proposals/fledge-multiple-seller-testing/README.md [hunk 1]
```diff
@@ -8,13 +8,13 @@ To enable early FLEDGE testing for different exchanges on Ad Manager inventory d
...
+Sellers might need to update their previously provided auction configuration for auctions run when [refreshing an ad slot](https://developers.google.com/publisher-tag/samples/refresh). To support auction configuration updates, later calls with the same `configKey` would replace the stored component `auctionConfig` for that seller and slot, while `Slot.setConfig({ componentAuctions: [{configKey, null}] })` would remove it.
 
 Component seller bids into the top-level auction will be interpreted by Ad Manager as “publisher payout net of fees or revshares” expressed in USD CPM units. Ad Manager will compare the bids of all sellers and the best contextual ad (which is selected via [dynamic allocation](https://support.google.com/admanager/answer/3721872?hl=en)), and will serve the ad with the highest bid.
 
 ## Next steps
-We’re working to have the `addComponentAuction` API available by early July and look forward to testing during the origin trial. 
+The experimental GPT method is now available. Please see the [GPT developer documentation](https://developers.google.com/publisher-tag/reference#googletag.config.componentauctionconfig) for further details.
```
Comment (sbelov) [inline proposals/fledge-multiple-seller-testing/README.md:18]: nit: perhaps, "experimental support in GPT"

File: proposals/fledge-multiple-seller-testing/README.md [hunk 2]
```diff
@@ -8,13 +8,13 @@ To enable early FLEDGE testing for different exchanges on Ad Manager inventory d
...
 
-Sellers might need to update their previously provided auction configuration for auctions run when [refreshing an ad slot](https://developers.google.com/publisher-tag/samples/refresh). To support auction configuration updates, later calls with the same `seller` would replace the stored component `AuctionConfig` for that seller and slot, while `Slot.addComponentAuction(seller, null)` would remove it.
+Sellers might need to update their previously provided auction configuration for auctions run when [refreshing an ad slot](https://developers.google.com/publisher-tag/samples/refresh). To support auction configuration updates, later calls with the same `configKey` would replace the stored component `auctionConfig` for that seller and slot, while `Slot.setConfig({ componentAuctions: [{configKey, null}] })` would remove it.
```
Comment (sbelov) [inline proposals/fledge-multiple-seller-testing/README.md:13]: s/for that seller/previously associated with that `configKey`

Comment (rahulkooverjee-google) [reply proposals/fledge-multiple-seller-testing/README.md]: Done

File: proposals/fledge-multiple-seller-testing/README.md (hunk 1 above)
Comment (rahulkooverjee-google) [reply proposals/fledge-multiple-seller-testing/README.md]: Done

**Review by sbelov — APPROVED** [review]

**Pull request merged**